ajd/Animal/lib/Animal.pm

```diff
@@ -48,6 +71,70 @@ sub sound {
   die 'You must define sound() in a subclass';
 }
 
+=head2 default_colour
+
+The default colour of an animal. This must be overidden.
+
+=cut
+
+sub default_colour {
+  die 'You must define default_colour() in a subclass';
```

Comment:
Didn't the book want the default colour to be brown across _all_ animals, unless overridden?